lib/pysquared/pysquared.py

```diff
@@ -12,6 +12,7 @@
 import time
 from collections import OrderedDict
 from os import chdir, mkdir, stat
+from typing import Optional
```

Comment:
Unfortunately the typing library is not available in circuitpython so we'll need to wrap this in a try/except. It looks like this can be added to the existing typing import on line 38.